### ISSUE TITLE
Skript doesn't follow order of operations with multiple divisors

### ISSUE THREAD

**Issue by bluelhf** [SkriptLang/Skript#2886]
### Description
The arithmetic expression `a / b / c` is treated as `a / (b / c)` instead of `(a / b) / c`

### Steps to Reproduce
Run the following script:
```vb
on script load:
  broadcast "Expected 10 / 5 / 7 (%(10 / 5) / 7%), got 10 / (5 / 7) (%10 / 5 / 7%)"
```
Output is `Expected 10 / 5 / 7 (0.29), got 10 / (5 / 7) (14)`.
Grab a calculator and calculate `10 / 5 / 7`. The result is `0.285714285714...` which rounds to `0.29`
Observe the incorrect result of Skript's normal arithmetic.

### Expected Behavior
The equation should be done correctly, according to the order of operations, and for `10 / 5 / 7`, 0.29 should be returned (provided the number accuracy is 2 in config.sk)

### Server Information
* **Server version/platform:** Paper 1.15.2 (build 143)
* **Skript version:** Skript 2.5-alpha2

**Comment by bluelhf** [comment]
Closed as duplicate of #535
Bumping that instead, it's medium priority and has no fix yet?

**Comment by Wealthyturtle** [comment]
@bluelhf keep in mind that using a 3rd party Math library is not necessarily “easier” as Skript is first and foremost still a scripting language. It would be complicated to rewrite the entire parser for integration with a 3rd party Math parser. What would happen to expressions or functions used in conjunction to arithmetic operations for example?
